firestore/google/cloud/firestore_v1/client.py

```diff
@@ -67,6 +70,11 @@ class Client(ClientWithProject):
         database (Optional[str]): The database name that the client targets.
             For now, :attr:`DEFAULT_DATABASE` (the default value) is the
             only valid database.
+        client_info (Optional[google.api_core.client_info.ClientInfo]):
```

Comment:
Should this be documented as a `google.api_core.gapic_v1.client_info.ClientInfo`, instead?

Same comment as https://github.com/googleapis/google-cloud-python/pull/7876#discussion_r281872712